k2/csrc/ragged_test.cu

```diff
@@ -128,6 +128,23 @@ TEST(RaggedTest, TestRaggedFromString) {
   }
 }
 
+TEST(RaggedTest, TestRaggedFromVecs) {
+  std::vector<std::vector<int32_t>> vecs;
```

Comment:
may be we can just write `vecs = {{7,9}, {10,12,13}`, not sure.

Reply:
It's ok. Will change to it.